mods/ts/sequences/structures.yaml

```diff
@@ -862,6 +862,10 @@ proc: # TODO: unused narefn_a, narefn_b
 	make: ntrefnmk
 		Length: 20
 		ShadowStart: 20
+	dock: ntrefn_a # TODO: unused
```

Comment:
That makes sense. Added it to the list at https://github.com/OpenRA/OpenRA/issues/7874.